miner/miner.go

```diff
@@ -353,8 +391,8 @@ func (m *Miner) computeTicket(ctx context.Context, addr address.Address, base *M
 	if err := addr.MarshalCBOR(buf); err != nil {
 		return nil, xerrors.Errorf("failed to marshal address to cbor: %w", err)
 	}
-
-	input, err := m.api.ChainGetRandomness(ctx, base.ts.Key(), crypto.DomainSeparationTag_TicketProduction, (base.ts.Height()+base.nullRounds+1)-1, buf.Bytes())
+	input, err := m.api.ChainGetRandomness(ctx, base.ts.Key(), crypto.DomainSeparationTag_TicketProduction,
+		base.ts.Height()+base.nullRounds+1-build.TicketRandomnessLookback, buf.Bytes())
```

Comment:
Aren't we supposed to mix beacon values into the ticket chain somewhere?

Reply:
no,
election only drand
post using ticket chain

Reply:
nope, apparently not